### PR TITLE
SSCS-4400 Evidence received MVP

### PR DESCRIPTION
**Pull request by oleg-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/SSCS-4400

### Change description ###
Evidence received letters for appellant, appointee and reps.

**Does this PR introduce a breaking change?**

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java [hunk 1]
```diff
@@ -437,6 +437,90 @@ private void validateLetterNotifications(List<String> expectedLetterTemplateIds,
...
+                Arrays.asList("7af36950-fc63-45d1-907d-f472fac7af06"),
+                Collections.emptyList(),
+                Arrays.asList("6af62d46-98e5-4ade-aa72-e4a11c56286e"),
+                "no",
+                "no",
+                "yes",
+                "no",
+                "1",
+                "0",
+                "1"
```
Comment (arcotc) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java]: According to the ticket, regardless of the subscription for the appellant, this letter should be sent to them.

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java [hunk 2]
```diff
@@ -437,6 +437,90 @@ private void validateLetterNotifications(List<String> expectedLetterTemplateIds,
...
+                Arrays.asList("cab48431-a4f0-41f5-b753-2cecf20ab5d4", "7af36950-fc63-45d1-907d-f472fac7af06"),
+                Collections.emptyList(),
+                Collections.emptyList(),
+                "yes",
+                "no",
+                "yes",
+                "no",
+                "2",
+                "0",
+                "0"
```
Comment (arcotc) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java]: According to the ticket, regardless of the subscription for the appellant/rep, this letter should be sent to them.

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java [hunk 3]
```diff
@@ -437,6 +437,90 @@ private void validateLetterNotifications(List<String> expectedLetterTemplateIds,
...
+                Arrays.asList("30260c0b-5575-4f4e-bce4-73cf3f245c2d"),
+                Collections.emptyList(),
+                Arrays.asList("6af62d46-98e5-4ade-aa72-e4a11c56286e"),
+                "no",
+                "no",
+                "yes",
+                "no",
+                "1",
+                "0",
+                "1"
```
Comment (arcotc) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java]: According to the ticket, regardless of the subscription for the appellant/rep, this letter should be sent to them.

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java [hunk 4]
```diff
@@ -437,6 +437,90 @@ private void validateLetterNotifications(List<String> expectedLetterTemplateIds,
...
+                Arrays.asList("c5654134-2e13-4541-ac73-334a5b5cdbb6", "30260c0b-5575-4f4e-bce4-73cf3f245c2d"),
+                Collections.emptyList(),
+                Collections.emptyList(),
+                "yes",
+                "no",
+                "yes",
+                "no",
+                "2",
+                "0",
+                "0"
```
Comment (arcotc) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java]: According to the ticket, regardless of the subscription for the appellant/rep, this letter should be sent to them.

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java [hunk 5]
```diff
@@ -1402,6 +1486,58 @@ private void validateLetterNotifications(List<String> expectedLetterTemplateIds,
...
+                EVIDENCE_RECEIVED_NOTIFICATION,
+                "paper",
+                Arrays.asList("c5654134-2e13-4541-ac73-334a5b5cdbb6"),
+                Collections.emptyList(),
+                Collections.emptyList(),
+                "yes",
+                "no",
+                "1",
+                "0",
+                "0",
```
Comment (arcotc) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java]: According to the ticket, regardless of the subscription for the appointee, this letter should be sent to them.

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java [hunk 6]
```diff
@@ -1402,6 +1486,58 @@ private void validateLetterNotifications(List<String> expectedLetterTemplateIds,
...
+                EVIDENCE_RECEIVED_NOTIFICATION,
+                "oral",
+                Arrays.asList("c5654134-2e13-4541-ac73-334a5b5cdbb6"),
+                Collections.emptyList(),
+                Collections.emptyList(),
+                "yes",
+                "no",
+                "1",
+                "0",
+                "0",
```
Comment (arcotc) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java]: According to the ticket, regardless of the subscription for the appointee, this letter should be sent to them.

File: src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java [hunk 7]
```diff
@@ -47,7 +47,7 @@
 @Component
 @Slf4j
 public class Personalisation<E extends NotificationWrapper> {
-    private static final List<NotificationEventType> FALLBACK_LETTER_SUBSCRIPTION_TYPES = Arrays.asList(APPEAL_LODGED, SYA_APPEAL_CREATED_NOTIFICATION);
+    private static final List<NotificationEventType> FALLBACK_LETTER_SUBSCRIPTION_TYPES = Arrays.asList(APPEAL_LODGED, SYA_APPEAL_CREATED_NOTIFICATION, EVIDENCE_RECEIVED_NOTIFICATION);
```
Comment (arcotc) [inline src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java]: According to the ticket, regardless of the subscription for the appellant, therefore this is not a FALLBACK letter, but rather a MANDATORY letter.

File: src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationValidService.java [hunk 8]
```diff
@@ -17,7 +17,7 @@
 
 @Service
 public class NotificationValidService {
-    private static final List<NotificationEventType> FALLBACK_LETTER_SUBSCRIPTION_TYPES = Arrays.asList(APPEAL_LODGED, SYA_APPEAL_CREATED_NOTIFICATION);
+    private static final List<NotificationEventType> FALLBACK_LETTER_SUBSCRIPTION_TYPES = Arrays.asList(APPEAL_LODGED, SYA_APPEAL_CREATED_NOTIFICATION, EVIDENCE_RECEIVED_NOTIFICATION);
```
Comment (arcotc) [inline src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationValidService.java]: According to the ticket, regardless of the subscription for the appellant, therefore this is not a FALLBACK letter, but rather a MANDATORY letter.

**Comment by oleg-hmcts** [comment]
Implementation as per clarified ACs.

**Pull request merged**